### PR TITLE
docs(radio-button, overlay-menu): update example usage

### PR DESCRIPTION
**Pull request by bualoy-napat**
## Description

Add example usage in the section of handle check value changed event

Fixes # (issue)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: documents/src/pages/elements/radio-button.md [hunk 1]
```diff
@@ -72,12 +72,39 @@ More than one `ef-radio-button` can be grouped by setting the same value to the
 ## Handle check value changed
 `checked-changed` is the **only** event fired by `ef-radio-button`. It is dispatched whenever the state has been changed by user interaction, such as a click, tap or keyboard event.
```
Comment (wattachai-lseg) [inline documents/src/pages/elements/radio-button.md:74]: Missing "Usually, apps would listen to `checked-changed`..." paragraph.

File: documents/src/pages/elements/overlay-menu.md [hunk 2]
```diff
@@ -155,13 +154,23 @@ Menu and sub-menus are bound together using the `for` and `id` attributes of `ef
 ::import-elements::
```
Comment (wattachai-lseg) [inline documents/src/pages/elements/overlay-menu.md:156]: missing content update for "Nested menus" section.

**Review by wattachai-lseg — CHANGES_REQUESTED** [review]

File: documents/src/pages/elements/radio-button.md [hunk 3]
```diff
@@ -72,12 +72,42 @@ More than one `ef-radio-button` can be grouped by setting the same value to the
...
 
+Usually, apps would listen to `checked-changed` event of multiple radio buttons in a group. This could be done
+by listening to the event at the wrapper element of radio buttons with `{ capture: true}`.
```
Comment (wattachai-lseg) [inline documents/src/pages/elements/radio-button.md]: Unnecessary new line.

Comment (bualoy-napat) [reply documents/src/pages/elements/radio-button.md]: Updated!

**Review by wattachai-lseg — APPROVED** [review]

**Review by wattachai-lseg — APPROVED** [review]

**Pull request merged**